src/node_main_instance.cc

```diff
@@ -175,8 +175,10 @@ std::unique_ptr<Environment> NodeMainInstance::CreateMainEnvironment(
 
   Local<Context> context;
   if (deserialize_mode_) {
-    context =
-        Context::FromSnapshot(isolate_, kNodeContextIndex).ToLocalChecked();
+    if (!Context::FromSnapshot(isolate_, kNodeContextIndex).ToLocal(&context)) {
+      *exit_code = 12;
```

Comment:
I think this may not be the right exit code? According to the documentation for exit code 12:

> `12` **Invalid Debug Argument** - The `--inspect` and/or `--inspect-brk` options were set, but the port number chosen was invalid or unavailable.

Reply:
I think 1 would be okay here, as an empty `.ToLocal()` essentially implies an uncaught exception of some kind

Reply:
Done.